manifests/plugin/dhcp/infoblox.pp

```diff
@@ -14,12 +14,18 @@
 #
 # $network_view:: The network view to use
 #
+# $wait_after_restart:: Number of seconds to wait after Grid restart
+#
+# $options:: DHCP options in JSON format: [{"name": "","num": 1,"value": "","vendor_class": ""}]
+#
 class foreman_proxy::plugin::dhcp::infoblox (
   String $username = undef,
   String $password = undef,
   Enum['host', 'fixedaddress'] $record_type = 'fixedaddress',
   String $dns_view = 'default',
   String $network_view = 'default',
+  Integer $wait_after_restart = 10,
+  String $options = undef,
```

Comment:
I am going to make this `'[]'`.

Reply:
Can you help me how to refactor this if I default to "[]" here:

```
<% unless [nil, :undefined, :undef].include?(scope.lookupvar('::foreman_proxy::plugin::dhcp::infoblox::options')) -%>
```

Reply:
Not 100% sure how this should be at the end... is the input a single string or an array of strings?

Reply:
Id the default is just '[]' as string any thing can be set, just render out the variable.

Reply:
Why do you use the String data type and not at least `Array[String]`? I'm not sure how Kafo would deal with `Array[Hash]`, but `Array[String]` should be fine.
```suggestion
  Array[String] $options = [],
```

Reply:
It is actually string with JSON, I will make it '[]' as the default.